### PR TITLE
Minutes for 2022/03/28

### PR DESCRIPTION
**Pull request by gregdyke**
## What does this change?

Module:
Week(s):

## Rendered Pages

-----
[View rendered docs/contributing/minutes.md](https://github.com/CodeYourFuture/syllabus/blob/gregdyke-patch-1/docs/contributing/minutes.md)

### CODE REVIEW

File: docs/contributing/minutes.md [hunk 1]
```diff
@@ -4,6 +4,103 @@ title: Team Minutes
...
+Week 1: imaginary user stories for imaginary project
+Week 2: defining user stories for own website
+Week 2: choose a user story 
+Week 3: do some more of it
+
+We floated the idea of a "shop" page
+
+Step 1) make what we have make sense, let’s start with 
+
+@Greg: Make a ticket for making a project
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- @Greg: Make a ticket for making a project
```

File: docs/contributing/minutes.md [hunk 2]
```diff
@@ -4,6 +4,103 @@ title: Team Minutes
...
+@Daniel: we can make a shop -> make a ticket for it
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- @Daniel: we can make a shop -> make a ticket for it
```

File: docs/contributing/minutes.md [hunk 3]
```diff
@@ -4,6 +4,103 @@ title: Team Minutes
...
+@Sally: consolidate current version to make sense, based on improving the personal home page
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- @Sally: consolidate current version to make sense, based on improving the personal home page
```

Comment (SallyMcGrath) [reply docs/contributing/minutes.md]: erm, lol. Actually maybe it's quicker just to say: your line breaks aren't showing up in markdown so would it be ok if you either:

- write
- lists

Or use full stops and capital letters? It's a bit hard to read.

**Review by SallyMcGrath — APPROVED** [review]
it's only a minor point

File: docs/contributing/minutes.md (hunk 3 above)
Comment (alismith-dev) [reply docs/contributing/minutes.md]: I took the liberty of list-ifying some bits to make it easier to read when I was reviewing. Hope that's ok with you @gregdyke!

**Pull request merged**